plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1362,6 +1374,7 @@ dialog.about.message=<html><font style="font-size: 23px; font-weight:">\
   Hosted and maintained by Pylo (https://www.pylo.co).<br><br>\
   Click below to read MCreator and third-party licenses.<br><br><br>
 dialog.about.title=About MCreator
+dialog.about.latest_version_not_applicable=N/A
```

Comment:
Where is this used?

Reply:
It accidentally escaped from another PR xD

Reply:
:heavy_check_mark: Fixed
The string will soon return to its home PR :)